Ionic 2 Android build fails when plugin is added to project

<!--- Provide a general summary of the issue in the Title above -->
## Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->
- Plugin version: 2.2.4
- Platform: Android
- OS version: Linux Mint 18
- Device manufacturer and model: -
- Cordova version (`cordova -v`): 6.4.0
- Cordova platform version (`cordova platform ls`):
  Installed platforms:
  android 6.0.0
  Available platforms: 
  amazon-fireos ~3.6.3 (deprecated)
  blackberry10 ~3.8.0
  browser ~4.1.0
  firefoxos ~3.6.3
  ubuntu ~4.3.4
  webos ~3.7.0
- Plugin configuration options: -
- Link to your project: -

Issue details posted originally here: https://github.com/driftyco/ionic-native/issues/752 and Ionic team says it's this plugin's issue.

When I generate a new starter project:
`ionic start test-project --v2`
and add the [BackgroundGeolocation](http://ionicframework.com/docs/v2/native/backgroundgeolocation/) plugin from Ionic Native:
`ionic plugin add cordova-plugin-mauron85-background-geolocation`
the android build:
`ionic build android`
fails with this error:

```
:processDebugResources
/home/pavlina/Projects/ionic-2.rc-1/platforms/android/res/xml/authenticator.xml:5:19-33: AAPT: No resource found that matches the given name (at 'icon' with value '@drawable/icon').

/home/pavlina/Projects/ionic-2.rc-1/platforms/android/res/xml/authenticator.xml:6:24-38: AAPT: No resource found that matches the given name (at 'smallIcon' with value '@drawable/icon').

/home/pavlina/Projects/ionic-2.rc-1/platforms/android/build/intermediates/res/merged/debug/xml/authenticator.xml:2: error: Error: No resource found that matches the given name (at 'icon' with value '@drawable/icon').

/home/pavlina/Projects/ionic-2.rc-1/platforms/android/build/intermediates/res/merged/debug/xml/authenticator.xml:2: error: Error: No resource found that matches the given name (at 'smallIcon' with value '@drawable/icon').

FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 9.847 secs

Error: /home/pavlina/Projects/ionic-2.rc-1/platforms/android/gradlew: Command failed with exit code 1 Error output:
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
/home/pavlina/Projects/ionic-2.rc-1/platforms/android/res/xml/authenticator.xml:5:19-33: AAPT: No resource found that matches the given name (at 'icon' with value '@drawable/icon').

/home/pavlina/Projects/ionic-2.rc-1/platforms/android/res/xml/authenticator.xml:6:24-38: AAPT: No resource found that matches the given name (at 'smallIcon' with value '@drawable/icon').

/home/pavlina/Projects/ionic-2.rc-1/platforms/android/build/intermediates/res/merged/debug/xml/authenticator.xml:2: error: Error: No resource found that matches the given name (at 'icon' with value '@drawable/icon').

/home/pavlina/Projects/ionic-2.rc-1/platforms/android/build/intermediates/res/merged/debug/xml/authenticator.xml:2: error: Error: No resource found that matches the given name (at 'smallIcon' with value '@drawable/icon').

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

Please let me know how to fix this, as the plugin is not usable for me at the moment and I need it in my project. 

Thanks!

--- a/README.md
+++ b/README.md
@@ -83,8 +83,8 @@ You will need to ensure that you have installed the following items through the 
 | Google Repository          | 24      |
 
 
-#### Backwards Compatability
-Please note that as of Cordova Android 6.0.0 icons are by default in mipmap/ directory not drawable/, so this plugin will have a build issue on < 6.0.0 Cordova builds, you will need to update Authenticator.xml to drawble directory from mipmap directory to work on older versions.
+#### Backwards Compatibility
+Please note that as of Cordova Android 6.0.0 icons are by default in mipmap/ directory not drawable/ directory, so this plugin will have a build issue on < 6.0.0 Cordova builds, you will need to update Authenticator.xml to drawable directory from mipmap directory to work on older versions.
 
 
 ## Quick Example

--- a/README.md
+++ b/README.md
@@ -83,6 +83,10 @@ You will need to ensure that you have installed the following items through the 
 | Google Repository          | 24      |
 
 
+#### Backwards Compatability
+Please note that as of Cordova Android 6.0.0 icons are by default in mipmap/ directory not drawable/, so this plugin will have a build issue on < 6.0.0 Cordova builds, you will need to update Authenticator.xml to drawble directory from mipmap directory to work on older versions.
+
+
 ## Quick Example
 
 ```javascript

--- a/android/plugin/src/main/res/xml/authenticator.xml
+++ b/android/plugin/src/main/res/xml/authenticator.xml
@@ -2,6 +2,6 @@
 <account-authenticator
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:accountType="@string/account_type"
-    android:icon="@drawable/icon"
-    android:smallIcon="@drawable/icon"
+    android:icon="@mipmap/icon"
+    android:smallIcon="@mipmap/icon"
     android:label="@string/app_name"/>

